Program/Seasons.py

```diff
@@ -0,0 +1,220 @@
+import time
+
+
+
+#This is the stored pictures, calling "global" so the entire program can use them.
+setMediaPath("F:\COMP 120 Tinkering Graphics-JH182233\Pictures") 
+picfour = makePicture("Tree-Four-Seasons.jpg") 
+picwinter = makePicture("Tree-Winter.jpg") 
+picspring = makePicture("Tree-Spring.jpg") 
+picsummer = makePicture("Tree-Summer.jpg")
+picautumn = makePicture("Tree-Autumn.jpg")  
+picM = makeEmptyPicture(513, 556, black)
+pic = makeEmptyPicture(1025, 1110, black)
+
+
+#The main function for the program
+def main():
+#create an empty picture and populate it with others 
+    startX = 0
+    startY = 0 
+    repaint (pic)
+    time.sleep(1)  
+    copyInto(picspring, pic, startX, startY)
+    repaint (pic)
+    
+    startX = 512
+    startY = 0    
+    time.sleep(1) 
+    copyInto(picsummer, pic, startX, startY)
+    repaint (pic)
+    
+    startX = 0
+    startY = 554  
+    time.sleep(1)  
+    copyInto(picautumn, pic, startX, startY)
+    repaint (pic)
+    
+    startX = 512
+    startY = 554  
+    time.sleep(1)  
+    copyInto(picwinter, pic, startX, startY)
+    repaint (pic)
+    request()
+    
+    
+    
+#Now the program will ask for a season that you would like to view
+def request():
+    startX = 230
+    startY = 300   
+    season = requestString("What season would you like to activate?")
```

Comment:
Maybe include a comment making it clear that this request is expecting specific strings